From 74229d4a41fd807b152427883fc8738844b64bb2 Mon Sep 17 00:00:00 2001
From: Sam Lantinga <slouken@libsdl.org>
Date: Thu, 29 Dec 2022 14:52:28 -0800
Subject: [PATCH 122/458] Fixed keyboard scancode mapping for parenthesis

(thanks to @meyraud705 for tracking down the root cause!)

Fixes https://github.com/libsdl-org/SDL/issues/6787
Closes https://github.com/libsdl-org/SDL/pull/6937

(cherry picked from commit e1bd5bd071d066e7009261f96c896423f6fd8fd0)
---
 src/events/SDL_keysym_to_scancode.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/events/SDL_keysym_to_scancode.c b/src/events/SDL_keysym_to_scancode.c
index 3df4fd056..1783152de 100644
--- a/src/events/SDL_keysym_to_scancode.c
+++ b/src/events/SDL_keysym_to_scancode.c
@@ -238,8 +238,8 @@ static const Uint32 LinuxKeycodeKeysyms[] = {
     /* 176, 0x0b0 */    0x0, /* NoSymbol */
     /* 177, 0x0b1 */    0x1008FF78, /* XF86ScrollUp */
     /* 178, 0x0b2 */    0x1008FF79, /* XF86ScrollDown */
-    /* 179, 0x0b3 */    0x28, /* parenleft */
-    /* 180, 0x0b4 */    0x29, /* parenright */
+    /* 179, 0x0b3 */    0x0, /* NoSymbol */
+    /* 180, 0x0b4 */    0x0, /* NoSymbol */
     /* 181, 0x0b5 */    0x1008FF68, /* XF86New */
     /* 182, 0x0b6 */    0xFF66, /* Redo */
     /* 183, 0x0b7 */    0xFFCA, /* F13 */
-- 
2.17.1

